.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @ravil-mobile @krzysz00 @jerryyin @giuseros @manupak @sjw36 @yiqian1 @djramic @pcf000 @stefankoncarevic
```

Comment:
Let's add `/external/llvm-project` on the next line to indicate that we don't own upstream.

Reply:
If an external developer want to create a PR to our repo (either in /external or not), one of our "code owners" are still responsible for reviewing it. That's my understanding of how CODEOWNERS work, it is for poking right developers to do code reviews. I'd say we probably still want someone from our team to review. Since we haven't assigned code owners to that granularity, I propose that we just skip this for now.